src/components/Button/src/index.tsx

```diff
@@ -1,18 +1,17 @@
-import MaterialButton from '@material-ui/core/Button';
+import MaterialButton, { ButtonTypeMap } from '@material-ui/core/Button';
 import React from 'react';
 import BaseProps from '@gemeente-denhaag/baseprops';
+import { StylesProvider } from '@material-ui/core';
+import { classes } from './bem-mapping';
+import '@gemeente-denhaag/basestyles';
+import './mui-override.module.css';
+import './button.module.css';
 
 export interface ButtonProps extends BaseProps {
   /**
    * Simple click handler
    */
   onClick?: () => void;
-
-  /**
-   * Color for the component
-   */
-  color?: 'primary' | 'secondary' | 'default';
-
   /**
```

Comment:
Do we have a use-case for the href attribute here? From my personal point of view to create a good UX you need a clear visual distinction between buttons which perform actions on a page and links which redirect users to another page/piece of content. I always considered it bad design when a link is visually represented as a button, how do you feel about this? Is it necessary to have this attribute here? (@bddjong @Robbert)

Reply:
I agree that is not ideal UX, you could start with excluding the `href` attribute for starters, and consider it when the use case arises.

The nice thing about having the BEM class names, is that is easy to copy over the existing styling to a hypothetical Link Button component.

Reply:
@Robbert @mjcarsjens the button component has an `onClick` property which contains a method that, surprise surprise, gets called when the button is clicked. Wouldn't it be better to keep the Button's API consistent and do any and all expected functionality from this button click in the onClick method?

Specifying separate properties for each and every use case of a component can and will lead to an insanely bloated API in my opinion.

Reply:
I've removed the href